mediawiki-vue.json

```diff
@@ -0,0 +1,11 @@
+{
+	"extends": [
+		"./vue-es5.js"
+	],
+	"rules": {
+		"no-implicit-globals": "off",
```

Comment:
Is this really going to be needed for Vue? That's pretty crummy.

Reply:
It's not really needed, we could go without, but the no-implicit-globals rule is highly inconvenient for Vue files. As it is, it's already a bit silly especially in package modules, because RL wraps everything in closures already. In .vue files, the contents of the `<script>` tag get wrapped in a closure by RL (because `.vue` files can only be used in package modules), it doesn't make sense to force people to write IIFEs there.

Reply:
Fine.

Reply:
I think the rule also catches undeclared variables defined inside inner closures. The correct way to use this role would be to use sourceType:module, but that probably breaks other stuff.

Reply:
eslint does not allow `sourceType: 'module'` to be combined with `ecmaVersion: 5`, because modules don't exist in ES5

Reply:
However, if disabling `no-implicit-globals` also prevents us from catching other things, we might want to work around it in a different way, like avoiding top-level variables in .vue files wherever possible, and using `/* global foo */` where not.